ffi/tests/go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/ava-labs/firewood/ffi/tests
+
+go 1.23
+
+toolchain go1.23.6
+
+require (
+	github.com/ava-labs/firewood/ffi/v2 v2.0.0-20250218224920-ad24ac0dbc52
```

Comment:
How does this work? Why is this always "v2.0.0-20250218224920-ad24ac0dbc52"? Isn't that related to the commit hash?

Or is this then overridden by the `replace` line at the end of the file?

Reply:
this gets overridden by the replace at the end of the file. let me see if I can make this sthg else that's like a placeholder.